modules/cats-effect2/build.sbt

```
@@ -0,0 +1,21 @@
+import Dependencies.Version._
+import Utilities._
+
+name := "pureconfig-cats-effect2"
+
+crossScalaVersions := Seq(scala212, scala213)
+
+libraryDependencies ++= Seq(
+  "org.typelevel" %% "cats-effect" % "2.5.1"
+)
+
+developers := List(Developer("keirlawson", "Keir Lawson", "keirlawson@gmail.com", url("https://github.com/keirlawson")))
+
+osgiSettings
+
+OsgiKeys.exportPackage := Seq("pureconfig.module.catseffect.*")
```

This also needs to be updated.

```suggestion
OsgiKeys.exportPackage := Seq("pureconfig.module.catseffect2.*")
```